scripts/cwr-helpers.sh

```diff
@@ -194,21 +194,46 @@ function copy_xml() {
 
 
 function release_charm() {
-    charm_build_dir="$1"
-    lp_id="$2"
-    series="$3"
-    charm_name="$4"
-    channel="$5"
-
-    if [[ -n "$series" ]]; then
-        charm_id="cs:~$lp_id/$series/$charm_name"
-    else
-        charm_id="cs:~$lp_id/$charm_name"
-    fi
-
-    pushed_charm=`charm push $charm_build_dir $charm_id | head -1 | awk '{print $2}'`
-    charm release $pushed_charm --channel=$channel
-    charm grant $pushed_charm everyone --channel=$channel
+  # Push a directory to the charm store, release it and grant everyone access.
+  local charm_build_dir=$1
+  shift
+  local lp_id=$1
+  shift
+  local series=$1
+  shift
+  local charm_name=$1
+  shift
+  local channel=$1
+  shift
```

Comment:
I didn't know the shift built-in accepted a number. Do you want me to change it?